osbs/core.py

```diff
@@ -837,13 +837,14 @@ def import_image(self, name, stream_import):
 
         stream_import['metadata']['name'] = name
         for tag in imagestream_json.get('spec', {}).get('tags', []):
-            image_import = {
-                'from': tag['from'],
-                'to': {'name': tag['name']},
-                'importPolicy': tag.get('importPolicy'),
-                'referencePolicy': tag.get('referencePolicy'),
-            }
-            stream_import['spec']['images'].append(image_import)
+            if not tags or tag['name'] in tags:
```

Comment:
Minor suggestion is to reverse the check to `continue` the loop instead of nesting the main piece of code in the `if-block`. For example:

```python
if tags and tag['name'] not in tags:
  continue

# Do the work here
```

Reply:
will do